okhttp/src/main/java/okhttp3/ConnectionSpec.java

```diff
@@ -40,6 +40,17 @@
  */
 public final class ConnectionSpec {
 
+  // Most secure but supported list as defined by Google Cloud Platform.
+  // https://cloud.google.com/compute/docs/load-balancing/ssl-policies#profilefeaturesupport */
+  private static final CipherSuite[] RESTRICTED_CIPHER_SUITES = new CipherSuite[] {
+      CipherSuite.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
```

Comment:
Can you move the chacha suites below the aes suites so the order matches MODERN_TLS? My coworkers tell me Google loves chacha but they're the only ones.